hicommonwealth/commonwealth
# Added scroll to feed section + Updated Feed card design

## Link to Issue
Closes: https://github.com/hicommonwealth/commonwealth/issues/6537

## Description of Changes
- Updated dashboard page feed cards design 
- Updated dashboard page feed tabs to be sticky on scroll in mobile

## "How We Fixed It"
N/A

## Test Plan
- Visit dashboard page an login (make sure your account has some activity in some communities)
- Verify the feed cards in `For You` and `Global` tabs match [designs](https://www.figma.com/file/w6Tli9ApqBqgsZa7kB0Vk5/Mobile-ideation-Q4?type=design&node-id=761-52092&mode=dev)
- Verify when scrolling on mobile, the tabs section row ( `For You`, `Global` and `Chain`) becomes sticky when user scrolls past it.

## Deployment Plan
N/A

## Other Considerations
We can do a lot of refactor/cleanup in the dashboard components, that can be done if we want to include cleanup in this PR
